### DISCUSSION TITLE
Why using LastModifiedTime to sort our search result will not work

### DISCUSSION THREAD

**Discussion by mvcsharepointdev** [microsoft-search/pnp-modern-search#2852] [Q&A]
I define the following LastModifiedTime to sort my search results:-

![image](https://user-images.githubusercontent.com/79726511/225255430-ca6346d6-4f4c-4371-af86-62eaaf56fd47.png)

but the search results will not be sorted accordingly. any advice? For example i got a file which was modified on 9 march before 14 march, as follow

![image](https://user-images.githubusercontent.com/79726511/225256174-a9f4136c-cc3a-4ed7-830b-6747a487243e.png)

**Comment by kasperbolarsen** [comment]
And you have checked that LastModifiedTime is Selected in the Results web part?

**Reply by mvcsharepointdev** [reply]
@kasperbolarsen yes it is selected

**Comment by gretchunkim** [comment]
I'm also trying to troubleshoot MS Search data source sort issue.  @mvcsharepointdev  Are you using SP as data source or something else?  and which version are you using for PnP?

**Reply by mvcsharepointdev** [reply]
@gretchunkim yes i am using SP as the data source. thanks

**Comment by wobba** [comment]
The default cards show Created date, not last modified. You need to change the slot to use the modified date property instead.
<img width="1116" alt="image" src="https://user-images.githubusercontent.com/51104/225581780-0fc9d01a-c3c7-498e-bd6a-705c1c2f43b8.png">
